test/gtest/ucs/test_sys.cc

```diff
@@ -137,3 +184,25 @@ UCS_TEST_F(test_sys, memunits_to_str) {
     test_memunits(UCS_TBYTE, "1T");
     test_memunits(UCS_TBYTE * 1024, "1024T");
 }
+
+UCS_TEST_F(test_sys, memcpy) {
+    const double diff = 0.95; /* allow 5% fluctuations */
+    size_t size;
+    double memcpy_bw;
+    double memcpy_relax_bw;
+    char buf[256];
+    int i;
+
+    for (size = 4096; size <= 256 * UCS_MBYTE; size *= 2) {
+        ucs_memunits_to_str(size, buf, sizeof(buf));
+        UCS_TEST_MESSAGE << buf;
+        for (i = 0; i < 256; i++) {
```

Comment:
need to use ucs::perf_retry count, and retry if test failed (to avoid failures due to temporary load)
pls see test_profile for example

Reply:
ok, perf_retry_count is number of iterations, test breaks on success